tests/internal/test_fio_fillup.py

```diff
@@ -0,0 +1,62 @@
+"""
+Module to perform IOs with several weights
+"""
+import pytest
+import logging
+
+from ocs_ci.framework.testlib import BaseTest
+from ocs_ci.ocs import constants
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestPVCFillup(BaseTest):
```

Comment:
I'm not sure if we need this test except for the PR validation job, but I don't see a harm in having it either.